src/types.ts

```diff
@@ -74,6 +74,7 @@ interface BaseSocialStoriesShareSingleOptions extends Omit<BaseShareSingleOption
 
 export interface InstagramStoriesShareSingleOptions extends BaseSocialStoriesShareSingleOptions {
   social: Social.InstagramStories;
+  appId: string;
```

Comment:
Can somebody add more info here? What appId? Should this be the Facebook App Id? Or the AppStore app id?

Reply:
According to [Facebook's docs](https://developers.facebook.com/docs/instagram/sharing-to-stories/) on sharing to Instagram stories, the id is the Facebook app id.